deps/oblib/src/common/rowkey/ob_rowkey.h

```diff
@@ -189,7 +189,7 @@ class ObRowkey {
     if (obj_ptr_ == rhs.obj_ptr_) {
       cmp = static_cast<int32_t>(obj_cnt_ - rhs.obj_cnt_);
     } else {
-      cmp = compare_prefix(rhs);
+      compare_prefix(rhs, cmp);
```

Comment:
need check return code here. if you ignore the return code on purpose, you need write a comment to explain it or add a TODO task.

Reply:
Hi @simonjoylet I have added some comment.